mcloud.yaml

```diff
@@ -10,7 +10,7 @@ integrations:
     git_branch: $UV_BRANCH
     pip_install: poetry==1.7.1
 command: >-
-  cd ultravox && poetry install --no-dev && poetry run torchrun --nproc_per_node=8 -m ultravox.training.train $TRAIN_ARGS
+  cd ultravox && poetry install --no-dev && poetry run python -m ultravox.training.helpers.prefetch_weights $TRAIN_ARGS && poetry run torchrun --nproc_per_node=8 -m ultravox.training.train $TRAIN_ARGS
```

Comment:
Can we break this command into multiple lines for improved readability:

```
 cd ultravox &&
 poetry install --no-dev &&
 poetry run python -m ultravox.training.helpers.prefetch_weights $TRAIN_ARGS &&
 poetry run torchrun --nproc_per_node=8 -m ultravox.training.train $TRAIN_ARGS
 ```

Reply:
Worked out of the box. Nice!